docs/shared-content-source/docs/modules/develop/pages/cloudflow-configuration-kubernetes.adoc

```diff
@@ -0,0 +1,363 @@
+
+:page-partial:
+
+include::ROOT:partial$include.adoc[]
+
+== Available Kubernetes Settings 
+
+In this section we are covering the different Kubernetes parameters that are configurable. 
+
+To set the configuration on a container related to Kubernetes is necessary, first of all, to add that configuration under `cloudflow.streamlets.[streamlet-name].kubernetes`.
+On the followint samples we show configurations under a specific streamlet, `my-streamlet`, that in its complete path is `cloudflow.streamlets.my-streamlet`. It is also possible, as explained in xref:develop:cloudflow-configuration.adoc#_configuring_a_runtime_using_the_runtime_scope[Configuring a Runtime using the runtime Scope], to apply any configuration shown in here to any `runtimes`. The complete path for a runtime is, for instance, `cloudflow.runtimes.akka`.
+
+=== Resources and Environment Variables
```

Comment:
Please create separate sections for configuring resources and configuring environment variables. 
"Configure container resource requests and limits" could be a title for one, "Configure container environment variables" for the other.